components/os/component_output.go

```diff
@@ -448,11 +450,11 @@ func CreateGet(cfg Config, eventsStore events_db.Store) func(ctx context.Context
 
 		o := &Output{}
 
-		cctx, ccancel := context.WithTimeout(ctx, 10*time.Second)
-		virtEnv, err := pkg_host.SystemdDetectVirt(cctx)
```

Comment:
I read the function `pkg_host.SystemdDetectVirt`, it used Process. Should we append command into error message or log the command when getting error?

Reply:
done